test/test_ensemble_builder/test_manager.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from typing import Callable
+
+from autosklearn.automl import AutoML
+from autosklearn.ensemble_building import EnsembleBuilderManager
+
+from pytest_cases import parametrize_with_cases
+
+import test.test_automl.cases as cases
+from test.conftest import DEFAULT_SEED
+
+
+@parametrize_with_cases("automl", cases=cases, has_tag="fitted")
+def case_real_runs(
+    automl: AutoML,
+    make_ensemble_builder_manager: Callable[..., EnsembleBuilderManager],
+) -> EnsembleBuilderManager:
+    """Uses real runs from a fitted automl instance"""
+    manager = make_ensemble_builder_manager(
+        backend=automl._backend,
+        metric=automl._metric,
+        task=automl._task,
+        dataset_name=automl._dataset_name,
+        seed=automl._seed,
+        logger_port=automl._logger_port,
+        random_state=DEFAULT_SEED,
+    )
+    return manager
```

Comment:
Just out of curiosity, where is this manager used later? I don't see right now where the test for this is located.

Reply:
No where, I should have some test that use it. Sorry, slipped the todos

Reply:
Just checking: did you add such a check?

Reply:
I'll do so today, there's not too much functionality to check as most of the heavy lifting is done inside `EnsembleBuilder`. The real runs didn't fail so I assume it's okay, meaning I could start the benchmarking and write the tests after, almost nothing has changed. Hopefully no bugs appear in the testing